napari/_qt/qt_viewer.py

```diff
@@ -287,16 +287,19 @@ def _create_performance_dock_widget(self):
     def console(self):
         """QtConsole: iPython console terminal integrated into the napari GUI."""
         if self._console is None:
-            from .widgets.qt_console import QtConsole
+            try:
+                from napari_console import QtConsole
 
-            self.console = QtConsole({'viewer': self.viewer})
+                self.console = QtConsole(self.viewer)
+            except ImportError:
+                self._console = None
```

Comment:
maybe a warning with instructions on how to install the console?

Reply:
very nice idea

Reply:
I've added a warning, let me know if it was what you had in mind/ if you think text can be improved